### PR TITLE
DPS_hotfixes_1

### PR DESCRIPTION
**Pull request by ApurwaMohite**
Signed-off-by: ApurwaMohite <55824145+ApurwaMohite@users.noreply.github.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: config/test_dtm_config.yaml [hunk 1]
```diff
@@ -27,10 +27,13 @@ test_41230:
 
 test_42246:
   num_loop: 300
+  nclients: 2
```
Comment (priyankaborawake) [inline config/test_dtm_config.yaml:30]: @ApurwaMohite  Lets follow only clients, samples in config as well as test.

File: libs/dtm/dtm_recovery.py [hunk 2]
```diff
@@ -153,7 +152,7 @@ def perform_ops(self, workload_info: list, queue, skipread: bool = True,
                                            skip_write=True,
                                            skip_read=skipread,
                                            validate=validate,
-                                           log_file_prefix=f"read_workload_{workload['obj_size']}b",
```
Comment (priyankaborawake) [inline libs/dtm/dtm_recovery.py:156]: Please keep b in log file prefix.

File: tests/dtm/test_rgw_s3_process_restart.py [hunk 3]
```diff
@@ -240,8 +245,8 @@ def test_delete_during_rgw_svc_restart(self):
         self.log.info("Step 1: Perform WRITEs-READs-Validate Operations")
         self.dtm_obj.perform_write_op(bucket_prefix=f"bucket-{test_prefix}",
                                       object_prefix=f"object-{test_prefix}",
-                                      no_of_clients=self.test_cfg['clients'],
-                                      no_of_samples=self.test_cfg['samples'],
+                                      no_of_clients=test_cfg['nclients'],
```
Comment (priyankaborawake) [inline tests/dtm/test_rgw_s3_process_restart.py:248]: Please make the change for clients and samples.

File: libs/dtm/dtm_recovery.py (hunk 2 above)
Comment (ApurwaMohite) [reply libs/dtm/dtm_recovery.py:156]: this is adding extra b in logs, if we use workload list from 0b-512Mb. 
For other workload lists, do we have to add it here for log file?

File: tests/dtm/test_rgw_s3_process_restart.py [hunk 4]
```diff
@@ -426,8 +431,9 @@ def test_continuous_read_during_rgw_s3_restart(self):
         self.dtm_obj.perform_write_op(bucket_prefix=self.bucket_name,
                                       object_prefix=self.object_name,
                                       no_of_clients=self.test_cfg['clients'],
-                                      no_of_samples=self.test_cfg['test_42247']['nsamples'],
-                                      log_file_prefix=log_file_prefix, queue=que)
+                                      no_of_samples=self.test_cfg['test_42247']['samples'],
+                                      log_file_prefix=log_file_prefix, queue=que,
+                                      loop=self.test_cfg['loop_count'])
```
Comment (priyankaborawake) [inline tests/dtm/test_rgw_s3_process_restart.py:436]: @ApurwaMohite We were reading the same data in multiple loops. hence had added loop count in reads.

**Review by priyankaborawake — APPROVED** [review]
Minor fix required. 
approving changes.

**Review by kulrk — APPROVED** [review]

**Review by ravindrachoudhari — APPROVED** [review]

**Review by kachhwahadivya — APPROVED** [review]

**Comment by ApurwaMohite** [comment]
collection log:
[PR_1694.log](https://github.com/Seagate/cortx-test/files/8997964/PR_1694.log)

**Pull request merged**